src/api/todos.ts

```diff
@@ -5,4 +5,8 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export type TodoData = Pick<Todo, 'title' | 'completed' | 'userId'>;
+
+export const createTodo = async ({ title, completed, userId }: TodoData) => {
+  return client.post<Todo>('/todos', { title, completed, userId });
```

Comment:
you write same long code twice. and you will add when the parameters change.
```suggestion
export const createTodo = async (params: TodoData) => {
  return client.post<Todo>('/todos', params);
```